docs/src/main/asciidoc/security-basic-authentication-howto.adoc

```diff
@@ -15,23 +15,37 @@ Enable xref:security-basic-authentication.adoc[Basic authentication] for your Qu
 
 == Prerequisites
 
-* You have installed at least one extension that provides an `IdentityProvider` based on username and password, such as xref:security-jdbc.adoc[Elytron JDBC].
+* You have installed at least one extension that provides an `IdentityProvider` based on username and password.
+For example:
+
+* xref:security-properties[Elytron security properties file extension `(quarkus-elytron-security-properties-file)`]
+* xref:security-jdbc.adoc[Elytron security JDBC extension `(quarkus-elytron-security-jdbc)`]
+
+The following procedure outlines how you can enable Basic authentication for your application by using the `elytron-security-properties-file` extension.
 
 == Procedure
 
-. Enable Basic authentication by setting the `quarkus.http.auth.basic` property to `true`.
+. In the `application.properties` file, set the `quarkus.http.auth.basic` property to `true`.
 +
 [source,properties]
 ----
 quarkus.http.auth.basic=true
 ----
 
-. For testing purposes, you can configure the required user credentials, user name, secret, and roles, in the `application.properties` file.
+. To enable authentication for the embedded realm, set the `quarkus.security.users.embedded.enabled` property to `true`.
++
+[source,properties]
+----
+security.users.embedded.enabled=true
```

Comment:
Sorry Michelle but I'm not sure we need to highlight this property at all, I don't recall ever using it for tests

Reply:
It is enabled by default I guess, so may be below it works but if if it is enabled by default then it is 1 extra property to configure

Reply:
Sorry, I use it here https://github.com/quarkusio/quarkus/blob/main/integration-tests/csrf-reactive/src/main/resources/application.properties#L6 :-) but I'd not highlight in isolation, below it is fine

Reply:
Sorry for the noise, if you'd like to highlight then it is fine, but IMHO it  it should all be again done below, otherwise it reads as if it can be used outside of the testing, which it technically can be but we don't recommend it

Reply:
The default is false - see https://quarkus.io/guides/all-config#quarkus-elytron-security-properties-file_quarkus.security.users.embedded.enabled

I think it's fine to have the property just in the example bellow. IMHO no need to highlight this property in this separate part

Reply:
+1 to @rsvoboda's suggestion.

Reply:
@sberyozkin , @rsvoboda , @jsmrcka 

Thank you for your input and reviews on this bit too.

I think we have to keep in mind that the person reading this introductory-level how-to topic is likely to be new to Quarkus and is therefore likely 'playing' in a test environment. So while we don't recommend it for production, +1 to keeping it in and being consistent elsewhere.

I made some adjustments to the procedure to highlight that some configuration steps are purely for testing purposes.  How does this look now?

![image](https://github.com/quarkusio/quarkus/assets/92924207/33ad8783-a3bc-4a39-9f57-407c9e5ae772)

Reply:
@rsvoboda & @jsmrcka - I also propose that we diverge in the product version a little and either:
- Add a support note to the prereq section to highlight the extensions that we don't support
- Remove the extension altogether from the list in the product version

( @tqvarnst @inoxx03 & @sraghupu@redhat.com FYI)

Reply:
prereq section should be sufficient, otherwise we would need to drop all 3 mentioned extensions :)